ddl/ddl_api.go

```diff
@@ -509,13 +550,36 @@ func checkTooLongColumn(colDefs []*ast.ColumnDef) error {
 	return nil
 }
 
+func checkTooLongColumnName(colNames []*ast.ColumnName) error {
+	for _, colName := range colNames {
+		if len(colName.Name.O) > mysql.MaxColumnNameLength {
+			return ErrTooLongIdent.Gen("too long column %s", colName.Name)
```

Comment:
Should we add len(colName.Name.O) and MaxColumnNameLength  in the error message?

Reply:
The error message is consistent with CreateTableStmt